src/main/kotlin/no/nav/klage/oppgave/api/controller/KlagebehandlingDokumentController.kt

```diff
@@ -76,77 +79,6 @@ class DokumentController(
         )
     }
 
```

Comment:
Jeg mistenker at å miste togglingen kan skape litt utfordringer, men vi får se.

Reply:
Den er i og for seg enkel å ta inn igjen. Men det bør vel ikke være sånn at tilknyttede dokumenter både er i den store blobben med editerbare felter, og samtidig kan endres i toggling? Så her må nok dere frontendere bli enige om hva dere ønsker dere. :)

Reply:
Vi legger opp til å ha med `tilknyttedeDokumenter` i requesten til APIet når vi mob-programmerer dette i hvert fall.
Toggle skal ikke bli brukt i tillegg tenker jeg.